meson_options.txt

```
@@ -9,3 +9,7 @@ option('docdir', type : 'string', value : 'share/doc/rmw',
 
 option('without-curses', type : 'boolean', value : 'false',
        description : 'build without curses/menu support')
+
+option('fish-completions', type: 'boolean', value: true,
```

```suggestion
option('install_shell_completions', type: 'boolean', value: true,
```

I think I'd rather not have an option for each shell completion file.